src/RegistryInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Part of the Joomla Framework Registry Package
+ *
+ * @copyright  Copyright (C) 2023 Open Source Matters, Inc.
+ * @license    GNU General Public License version 2 or later; see LICENSE
+ */
+
+namespace Joomla\Registry;
+
+/**
+ * Registry Interface
+ *
+ * @method exists(string $key)
+ * @method get(string $key, $default = null)
+ * @method set(string $key, $value)
+ * @method remove(string $key)
+ *
+ * @since  2.1.0
+ */
+interface RegistryInterface {}
```

Comment:
Why empty? You can add here the get and set functions.

Reply:
Unfortunately we cannot, because `set()` method in Registry, will be incompatible:
https://github.com/joomla-framework/registry/blob/a3932a478dab2d7c38e65d91d185c6a86b32cc66/src/Registry.php#L453